libs/ui/src/ui_strings/audio_context.tsx

```diff
@@ -43,6 +43,19 @@ export interface UiStringsAudioContextProviderProps {
   children: React.ReactNode;
 }
 
+let webAudioContext: AudioContext | undefined;
+function getWebAudioContextInstance() {
+  if (!window.AudioContext) {
+    return undefined;
+  }
+
+  if (!webAudioContext) {
+    webAudioContext = new AudioContext();
+  }
```

Comment:
If one were inclined to play code golf, they might do this:
```suggestion
  webAudioContext ??= window.AudioContext ? new AudioContext() : undefined;
```

Or this, if that's too much:
```suggestion
  if (!window.AudioContext) {
    return undefined;
  }

  webAudioContext ??= new AudioContext();
```

Reply:
Honestly very tempted, but not enough to do the circleci roundtrip 😆